django/src/rdwatch/tasks/__init__.py

```diff
@@ -641,3 +661,38 @@ def generate_image_embedding(id: int):
     except Exception as e:
         # Handle exceptions (e.g., logging, showing error messages)
         print(f'Error processing image {site_image.id}: {e}')
+
+
+@signals.worker_ready.connect
+def download_sam_model_if_not_exists(**kwargs):
+    file_path = '/data/SAM/sam_vit_h_4b8939.pth'
```

Comment:
Can you make this a [`PathValue`](https://django-configurations.readthedocs.io/en/stable/values.html#configurations.values.PathValue) in settings.py instead? That way we a) have this defined in a single place, and b) someone running celery "natively" can choose a different location on disk for the pth file.